[CI] ApiKeyBackwardsCompatibilityIT testCreatingAndUpdatingApiKeys failing

**Build Scans:**
- [elasticsearch-intake #20465 / 7.17.29_bwc-snapshots](https://gradle-enterprise.elastic.co/s/wny4epsxvel3i)
- [elasticsearch-intake #20459 / 7.17.29_bwc-snapshots](https://gradle-enterprise.elastic.co/s/w3d6vjbistvt4)

**Reproduction Line:**
```
./gradlew ":x-pack:qa:rolling-upgrade:v7.17.29#oneThirdUpgradedTest" -Dtests.class="org.elasticsearch.upgrades.ApiKeyBackwardsCompatibilityIT" -Dtests.method="testCreatingAndUpdatingApiKeys" -Dtests.seed=4AA6B82E19FE0977 -Dtests.bwc=true -Dtests.locale=rof -Dtests.timezone=Asia/Pyongyang -Druntime.java=24
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.upgrades.ApiKeyBackwardsCompatibilityIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testCreatingAndUpdatingApiKeys),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: a string containing "all nodes must have version [8.10.0-8.10.4] or higher to support remote indices privileges for API keys"
     but: was "method [POST], host [http://127.0.0.1:38903], URI [_security/api_key/_bulk_update], status line [HTTP/1.1 400 Bad Request]
{"error":{"root_cause":[{"type":"action_request_validation_exception","reason":"Validation Failed: 1: remote index privileges cannot contain privileges that grant access to the failure store;"}],"type":"action_request_validation_exception","reason":"Validation Failed: 1: remote index privileges cannot contain privileges that grant access to the failure store;"},"status":400}"
```

**Issue Reasons:**
- [8.x] 2 failures in test testCreatingAndUpdatingApiKeys (1.1% fail rate in 175 executions)
- [8.x] 2 failures in step 7.17.29_bwc-snapshots (5.7% fail rate in 35 executions)
- [8.x] 2 failures in pipeline elasticsearch-intake (13.3% fail rate in 15 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/ApiKeyBackwardsCompatibilityIT.java
+++ b/x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/ApiKeyBackwardsCompatibilityIT.java
@@ -393,6 +393,8 @@ public class ApiKeyBackwardsCompatibilityIT extends AbstractUpgradeTestCase {
 
     private static RoleDescriptor randomRoleDescriptor(boolean includeRemoteDescriptors) {
         final Set<String> excludedPrivileges = Set.of(
+            "read_failure_store",
+            "manage_failure_store",
             "cross_cluster_replication",
             "cross_cluster_replication_internal",
             "manage_data_stream_lifecycle"

